Makefile

```diff
@@ -40,6 +40,11 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
+.PHONY: test-sec
+test-sec:
+	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
+	gosec -severity medium --confidence medium -quiet ./...
```

Comment:
this belongs into github.com/openshift/library-go/alpha-build-machinery

Reply:
@tnozicka ^^

Reply:
@tnozicka any feedback?

Reply:
@tnozicka ^^^ ping

Reply:
If we decide to use the tool, library-go should be the location.
I admit I don't have any experience with the tool but sounds like a linter. If we hook it up into makefile and nobody uses it it will just rot. We can't hook it into CI because of false positives. For the same reason we don't run golint.

Reply:
static analysis tools tend to give false positives. Hence why I'm not proposing this tool for CI, but for local testing. perhaps at some point it could be part of a common lint target or something of the sort.